shared/settings/invites/index.desktop.js

```diff
@@ -114,9 +114,9 @@ function PendingURLContent ({invite}: {invite: PendingURLInvite}) {
 function AcceptedInviteItem ({invite, onClick}: {invite: AcceptedInvite, onClick: (username: string) => void}) {
   const nameColor = stateColors(invite.currentlyFollowing, invite.trackerState, globalColors.blue).username
   return (
-    <Box style={{...styleInviteItem, ...globalStyles.clickable}} onClick={onClick}>
+    <Box style={{...styleInviteItem, ...globalStyles.clickable, flexShrink: 0}} onClick={onClick}>
       <Avatar
-        url={invite.avatar}
+        username={invite.username}
```

Comment:
Yeah, we don't get one now, just uid/username.  In the future we'll want to identify the uid we get back, so that we can find out whether they're followed and show that -- once we do that we'll have access to an avatar url.